### ISSUE TITLE
Crash report (GL)

### ISSUE THREAD

**Issue by dalu** [rotators/foclassic#43]
https://github.com/cvet/fonline/issues/58

d3d client has weird looking sprites for brahmin, human etc.

I have now changed resolution to 1280x1024, then back to 1024x768 which allowed me to log in and play again. (GL client)

**Comment by cvet** [comment]
Current engine doesn't support d3d rendering. Ask 8n different place.

**Comment by cvet** [comment]
Oh wait, there is foclassic branch, sorry...

**Comment by dalu** [comment]
yeah my bad I 1st posted in your repo because I confused something, then to not have to copy paste everything again posted a reference.

What I mean to say is:
The, opengl client, crashed twice today
and additionally, unrelated,
the d3d client only shows fire sprites instead of characters or brahmin, probably every sprite uses those.
why d3d client? because there is another issue open in this repo where the issue creator was asked if the crash happens in the d3d client. My answer to that would be "no, but... fire spirtes". :)
